README.md

```diff
@@ -22,6 +22,10 @@ When instantiating the plugin, your `opts` need the correct fields.
   "account": "can be anything; only used for logging",
   "token": "channel name in MQTT server",
   "limit": "amount of negative balance that can exist",
+  "warnLimit": "amount of negative balance before warning to settle",
+  "max": "amount of positive balance that can exist",
```

Comment:
Seems strange to have `limit` and `max`, instead of something like `min` and `max` or `lowLimit` and `highLimit`.

A wise man (@emschwartz) once said: Afraid of longer variable names, do not be. More time in the long run, they save.